_includes/manuals/3.1/1.2_release_notes.md

```diff
@@ -61,6 +61,61 @@ Existing installations should start thinking about a migration plan.
 
 Note that this is only support to run Foreman and Foreman Proxy themselves on EL7. Managing EL7 systems remains supported. See [the RFC](https://community.theforeman.org/t/deprecation-plans-for-foreman-on-el7-debian-10-and-ubuntu-18-04/25008) for more information.
 
+### Release notes for 3.1.1
+#### Foreman - Audit Log
+* Recent audits card in new host page is missing username \- [#34223](https://projects.theforeman.org/issues/34223)
+
+#### Foreman - Compute resources - VMware
+* Unable to create compute profile for specific vmware compute resource \- [#30550](https://projects.theforeman.org/issues/30550)
+* Cannot view Compute-Profiles from a specific Compute-Resource \- [#28155](https://projects.theforeman.org/issues/28155)
+
+#### Foreman - Facts
+* New OS created due to facts mismatch for operatingsystem for RHSM, Puppet and Ansible \- [#33981](https://projects.theforeman.org/issues/33981)
+
+#### Foreman - Host creation
+* Host form redirect relies on the primary interface being the first interface \- [#34300](https://projects.theforeman.org/issues/34300)
+* Remove workaround for root_password encoding \- [#34289](https://projects.theforeman.org/issues/34289)
+
+#### Foreman - Host registration
+* Missing example for "Install packages" field \- [#34252](https://projects.theforeman.org/issues/34252)
+* Grammatical errors with Insecure help text at Host Registration \- [#34250](https://projects.theforeman.org/issues/34250)
+* Host facts are not uploaded to Foreman when host is registered with global registration \- [#34249](https://projects.theforeman.org/issues/34249)
+* Fix bool params in global registration template \- [#34206](https://projects.theforeman.org/issues/34206)
+
+#### Foreman - Monitoring
+* Report disallowed metric labels as "other" \- [#34122](https://projects.theforeman.org/issues/34122)
+
+#### Foreman - Packaging
+* Pin psych &lt; 4 \- [#34212](https://projects.theforeman.org/issues/34212)
```

Comment:
is that really noteworthy?

Reply:
The script generated it. If we don't want it, we should unset the Fixed in version.

Reply:
I don't mind too much :)